HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,527 @@
+
+## Rationale
+
+This token metadata standard has been developed with input from multiple developers and projects in the NFT space. The goal is to provide a more granular NFT standard for "Web3 NFTs", providing a more concrete implementation for HIP-412. 
+
+This HIP focuses on:
+
+1. Providing strict guidelines for artists to define their NFT token metadata
+2. Maintaining flexibility via the `properties` object allowing for non-specified fields
+3. Maintaining a good level of interoperability and coherence with other standards such as OpenSea
+4. Providing sufficient examples for all types of NFTs to find the correct metadata implementation as an artist
+  
+## Specification
+
+Below is the human-readable schema, presented to maximize clarity. 
+
+```json
+{
+    "name": "NFT name (required)",
+    "creator": "artist (required)",
+    "creatorDID": "DID URI (optional)",
+    "description": "human readable description of the asset (recommended)",
+    "image": "cid or path to the NFT's preview image file (required)",
+    "image_integrity": "SHA-256 digest of the file pointed by the image field (recommended)",
+    "type": "MIME type (required)",
+    "format": "standard specification (required - i.e. none, opensea, HIPXXX, hashaxis)",
+    "properties": {
+        // json object that cover the overarching properties of the token
+        "files": [
+            {
+                "uri": "uri to file (required)",
+                "sha256_checksum": "cryptographic hash of the representation of the resource the author expects to load (recommended)",
+                "type": "MIME type (required)",
+                "is_default_file": "(Type: boolean) indicates if the file is the main file for this NFT (conditionally optional - required when multiple files are listed)",
+                "metadata": "metadata object (optional)",
+                "metadata_uri": "uri to metadata (optional)",
+                "localization": [
+                    // optional localization array
+                    {
+                        "uri": "uri to file (required)",
+                        "locale": "language identifier (required)"
+                    }
+                ]
+            }
+        ],
+        "collection": "Define your collection name to which the NFT belongs (optional)",
+        "external_url": "External URI pointing to an informational page about your collection or specific NFT information i.e. www.mynft/collection/<serial>/ (optional)"
+    },
+    "attributes": [
+        // Can only contain trait types for rarity calculation
+        {
+            "trait_type": "name of trait (required)",
+            "value": "value for this trait (required)",
+            "max_value": "maximum possible value for this trait (optional)"
+        }
+    ]
+    // no additional root-level properties allowed - properties array should contain additional properties you may require
+}
+```
+
+Below is the field-specific rationale.
```

Comment:
It would be useful to include the JSON Schema representation like we did in HIP 10 and 412 so that validators and parsers can make use of it.